packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -265,6 +267,8 @@ export const Tooltip = React.forwardRef(
         ) {
           tooltipElRef.current.hidePopover()
           setIsPopoverOpen(false)
+        } else {
+          setIsPopoverOpen(false)
```

Comment:
Fixes bug where overlay would stay "stuck" and not able to close via keyboard due to popOver state being open but it not actually being in the DOM